README.md

```diff
@@ -37,5 +37,5 @@ On table terminal run:
 On scoreboad terminal run:
 
     $ export SOL_MACHINE_ID="666a3d6a9d194a23b90a24573558d2f4"
-    $ export SOL_FLOW_MODULE_RESOLVER_CONFFILE=foosball/table/conf-gtk.json
+    $ export SOL_FLOW_MODULE_RESOLVER_CONFFILE=foosball/scoreboard/conf-gtk.json
```

Comment:
you should name the files so it applies automatically, right now sol-conffile.c will try to load `sol-flow-$APPNAME-$BOARD_NAME.json`, then `sol-flow-$APPNAME.json`, then `sol-flow-$BOARD_NAME.json`, then `sol-flow.json` (export `SOL_LOG_LEVEL=4` to check out the paths it tries).

Then we could have the gtk as the fallback (`sol-flow.json`), with board-specific in `sol-flow-intel-edison-rev-c.json` (see `board_detect.json`).